### PR TITLE
Update interest invokers introduction

### PR DESCRIPTION
**Pull request by lukewarlow**
To show that submit buttons and area elements are supported

### CODE REVIEW

File: site/src/pages/components/interest-invokers.explainer.mdx [hunk 1]
```diff
@@ -23,11 +23,12 @@ layout: ../../layouts/ComponentLayout.astro
 
 Following [invokers](../invokers.explainer), adding an `interesttarget`
 attribute to interactive elements: starting with `<button>`, `<input
-type="button">`/`<input type="reset">`, and `<a>` (perhaps expanding to
-`<area>`, `<input>`, `<textarea>`, `<select>`, `<summary>` or maybe more, see
+type="button">`/`<input type="reset">`/`<input type="submit">`, `<a>`
+ and `<area>` (perhaps expanding to
```
Comment (lukewarlow) [inline site/src/pages/components/interest-invokers.explainer.mdx:27]: Area is included further down the page so this just fixes that here. Submit buttons are similarly mentioned as supported so make that explicit here.

**Comment by keithamus** [comment]
I think in https://github.com/openui/open-ui/issues/839#issuecomment-1739848561 we only resolved to support Input/Button/Anchor right?

> * `RESOLVED: support interesttarget on buttons (<input type=button> or <button>) and links (<a>) only, and explore expanding to other element types in the future.`

**Comment by lukewarlow** [comment]
Ah correct hmm then I guess this should be updated the other way. 

I'm not keen on it being on anchor but not area personally though. If it's not an element we should be using we should deprecate it.

**Comment by lukewarlow** [comment]
Fixed that and made #1003

**Review by keithamus — APPROVED** [review]

**Comment by lukewarlow** [comment]
I apparently messed up the code owners change? Could you see what I did wrong?

**Review by mfreed7 — APPROVED** [review]

**Pull request merged**